cli/lib/nf3.mjs

```diff
@@ -222,6 +225,30 @@ class Nf3 {
     return res.data.address;
   }
 
+  /**
+   Sends an approve transaction to an ERC20/ERC721/ERC1155 contract for a certain amount of tokens
+   * @param {string} ercAddress - The Ethereum address of the transaction sender
+   * @param {string} tokenType  - Type of token 
+   * @param {string} value  - Amount of tokens to be approved (in Wei)
+   * @returns {Promise} transaction
+   */
+  async approve(ercAddress, tokenType, value) {
```

Comment:
What's the distinction between this function in `nf3.mjs` and the one in `tokens.mjs`? Do we need both?

Reply:
My mistake. Approve function in nf3.mjs should not have been committed.